tests/infra_test.go

```diff
@@ -665,117 +667,125 @@ var _ = Describe("Infrastructure", func() {
 				Transport: tr,
 			}
 
-			errors := make(chan error)
-			for ix := 0; ix < concurrency; ix++ {
-				go func(ix int) {
-					req, _ := http.NewRequest("GET", metricsURL, nil)
-					resp, err := client.Do(req)
+			for _, url := range metricsURLs {
+				if strings.Index(url, "[") != -1 {
```

Comment:
by this you are checking if the utl is ipv6 related?

Reply:
yes, because its a url, and we are adding "[" atm just for ipv6
I can split the https://ip:port/metric to see if the ip part is ipv6 (after stripping ipv6),
just thought we are going to drop it soon so didnt bother but will do if needed.

Reply:
I think it's best to be specific here, like you suggested. 
temporary changes end up to be the new const ones, so it's better to do it as best as we can.

Reply:
Done